src/commands/case_management.py

```diff
@@ -690,68 +694,61 @@ async def cmd_list(ctx: Context):
         for word in words:  # type: str
             if word.startswith("-"):
                 if flags_set:
-                    raise RuntimeError("invalid usage")  # FIXME: usage warning to user
+                    return await _list_invalid_usage(ctx)
                 flags = RescueRenderFlags.from_word(word)
                 flags_set = True
             else:
                 # platform or bust
                 if platform_filter_set:
-                    raise RuntimeError("invalid usage")  # FIXME: usage error
+                    return await _list_invalid_usage(ctx)
 
                 try:
                     platform_filter = Platforms[word.upper()]
                 except KeyError:
                     return await ctx.reply(f"unrecognized platform '{word.upper()}'")
 
     else:
-        raise RuntimeError  # FIXME: usage error
+        return await _list_invalid_usage(ctx)
     logger.debug(f"flags set:= {flags} \t platform_filter := {platform_filter}")
-    active_rescues: typing.List[Rescue] = []
-    inactive_rescues: typing.List[Rescue] = []
 
+    rescues: typing.List[Rescue] = []
     rescue_filter = functools.partial(_rescue_filter, flags, platform_filter)
-
-    # for each rescue that doesn't matches the filter
-    for rescue in itertools.filterfalse(rescue_filter, iter(ctx.bot.board.values())):  # type: Rescue
-        # put it in the right list
-        if rescue.active:
-            active_rescues.append(rescue)
-        else:
-            inactive_rescues.append(rescue)
-    format_specifiers = "c"
-    if flags.show_assigned_rats:
-        format_specifiers += "r"
-    if flags.show_uuids:
-        format_specifiers += "@"
-
-    if not active_rescues:
-        await ctx.reply("No active rescues.")
+    logger.debug(f"rescue_filter: {rescue_filter}")
+
+    for rescue in itertools.filterfalse(rescue_filter, iter(ctx.bot.board.values())):
+        logger.debug(f"rescue: {rescue}")
+        rescues.append(rescue)
```

Comment:
[non-blocking] Since we are not building two lists at this point, but rather only one; and we are refactoring this line anyways:
I would rather this be a list comprehension, and the collection of `rescues` be emitted as a single debug call.
```suggestion
    # build a collection of rescues that match the filters
    rescues = list(itertools.filterfalse(rescue_filter, iter(ctx.bot.board.values())))
    logger.debug("{} matching rescues, rescues :={!r}", len(rescues), rescues)
```